spec/system/sign_out_spec.rb

```diff
@@ -19,7 +19,7 @@ def cognito_stubs
   end
 
   scenario 'user signed out after inactiviy' do
-    Rails.configuration.session_expiry_in_minutes = 90.minutes
+    Rails.configuration.session_expiry_in_minutes = 60.minutes
```

Comment:
probably didn't need to need to change since it's just a test and doesn't need to correspond to the real config, but ok...